content/en/news/releases/1.7.x/announcing-1.7.4/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Announcing Istio 1.7.4
+linktitle: 1.7.4
+subtitle: Patch Release
+description: Istio 1.7.4 patch release.
+publishdate: 2020-10-27
+release: 1.7.4
+aliases:
+    - /news/announcing-1.7.4
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.7.3 and Istio 1.7.4
+
+{{< relnote >}}
+
+## Changes
+
+- **Improved** TLS configuration on sidecar server-side inbound paths to enforce TLS 2.0 version along with recommended cipher suites. This is disabled by default and can enabled by setting the environment variable `PILOT_SIDECAR_ENABLE_INBOUND_TLS_V2` to true.
+
+- **Added** ability to configure domain suffix for multicluster installation ([Issue #27300](https://github.com/istio/istio/issues/27300))
```

Comment:
nit: some descriptions end in periods, some do not. Thinking we should be consistent and have it for all of them?